.github/workflows/build_packages.yaml

```diff
@@ -419,11 +457,11 @@ jobs:
         push: ${{ github.event_name == 'release' && !github.event.release.prerelease }}
         pull: true
         no-cache: true
-        platforms: linux/amd64,linux/arm64
+        platforms: linux/${{ matrix.arch }}
         tags: ${{ steps.meta.outputs.tags }}
         labels: ${{ steps.meta.outputs.labels }}
         build-args: |
-          BUILD_FROM=ghcr.io/emqx/emqx-builder/5.0-3:${{ matrix.otp }}-alpine3.14
+          BUILD_FROM=ghcr.io/emqx/emqx-builder/5.0-3:${{ matrix.otp }}-${{ matrix.os }}
           RUN_FROM=alpine:3.14
```

Comment:
Do not hardcode.
Should be converted from `${{ matrix.os }}`

Reply:
we want to release docker images only based on alpine3.14

Reply:
otherwise fix L465 too ?

Reply:
Yes that is what I meant, we should fix L465 to prevent people bump the alpine version in matrix.otp but forget about L465.